### PR TITLE
journal article page

### PR DESCRIPTION
**Pull request by vivektripathi2020**
journal article page design

### CODE REVIEW

File: source/_patterns/02-molecules/accordion/accordion-head.twig [hunk 1]
```diff
@@ -11,14 +11,13 @@
 				aria-controls="collapse-{{id}}"
 			>
 				{% if accordionLeftIcon %}
-					<span class="{{accordionLeftIconBg}}">
```
Comment (vinothk-hw) [inline source/_patterns/02-molecules/accordion/accordion-head.twig:14]: Use span for same left and right side unique..

File: source/_patterns/03-components/accordion/accordion.twig [hunk 2]
```diff
@@ -8,6 +8,7 @@
 				accordionLeftIconBg: item.accordionLeftIconBg,
 				accordionLeftIcon: item.accordionLeftIcon,
 				accordionRightIconBg: item.accordionRightIconBg,
+				accordiantoggleicon : item.accordiantoggleicon,
```
Comment (vinothk-hw) [inline source/_patterns/03-components/accordion/accordion.twig:11]: class we should not use object. Please use generic class update style from parent component based on design

File: source/_patterns/03-components/site/_journal-article.scss [hunk 3]
```diff
@@ -6,30 +6,374 @@ $page: ".journal-article";
...
+         }       
+    }
+    & .article-seprator{
+        @extend .border-bottom;
+        margin-top: .688rem;
+    }
+}
+
+.right-toggle-icon-inner{
+    vertical-align: -.25rem;
```
Comment (vinothk-hw) [inline source/_patterns/03-components/site/_journal-article.scss:363]: -0.25rem not -.25rem

File: source/_patterns/03-components/site/journal-article.twig [hunk 4]
```diff
@@ -13,9 +13,7 @@
 				accordion_wrapper_id: accordion_wrapper_id
 			} only %}
 		{% else %}
-			{% include "@atoms/text/horizontal-rule.twig" with {
-				horizontal_class: "border-bottom my-0"
-			} only %}
+			<div class="article-seprator"></div>
```
Comment (vinothk-hw) [inline source/_patterns/03-components/site/journal-article.twig:16]: border sperate style from parent not separate style

**Review by vinothk-hw — CHANGES_REQUESTED** [review]
Please check the comments and update the same..

File: source/_patterns/02-molecules/accordion/accordion-head.twig (hunk 1 above)
Comment (vivektripathi2020) [reply source/_patterns/02-molecules/accordion/accordion-head.twig:14]: used span on both places within span and used classes

File: source/_patterns/03-components/accordion/accordion.twig (hunk 2 above)
Comment (vivektripathi2020) [reply source/_patterns/03-components/accordion/accordion.twig]: removed the object and added class

File: source/_patterns/03-components/site/_journal-article.scss (hunk 3 above)
Comment (vivektripathi2020) [reply source/_patterns/03-components/site/_journal-article.scss]: used zero where value is in points

File: source/_patterns/03-components/site/journal-article.twig (hunk 4 above)
Comment (vivektripathi2020) [reply source/_patterns/03-components/site/journal-article.twig]: used parent class for border removed separate style

**Pull request merged**